<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::project::ProjectEdit</class>
 <widget class="QWidget" name="gams::studio::project::ProjectEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="laBaseDir">
        <property name="text">
         <string>Base directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="edBaseDir"/>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="laWorkDir">
        <property name="text">
         <string>Working directory</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="laMainGms">
        <property name="text">
         <string>Main file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edName"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edWorkDir"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="laName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="bBaseDir">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons/icons.qrc">
          <normaloff>:/solid/folder-open-bw</normaloff>:/solid/folder-open-bw</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="bWorkDir">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../icons/icons.qrc">
          <normaloff>:/solid/folder-open-bw</normaloff>:/solid/folder-open-bw</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="laProjectFile">
        <property name="text">
         <string>Project file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edProjectFile"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="laPfFile">
        <property name="text">
         <string>Parameter file</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cbPfFile"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbMainFile"/>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="bGspSwitch">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../icons/icons.qrc">
          <normaloff>:/solid/new</normaloff>:/solid/new</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edProjectFile</tabstop>
  <tabstop>edName</tabstop>
  <tabstop>edWorkDir</tabstop>
  <tabstop>bWorkDir</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
